Tests/XCUITests/HistoryTests.swift

```diff
@@ -103,215 +98,214 @@ class HistoryTests: BaseTestCase {
         XCTAssertEqual(app.alerts.buttons["OK"].exists, false)
     }
 
-    func testRecentlyClosedOptionAvailable() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
-        navigator.goto(HistoryRecentlyClosed)
-        waitForNoExistence(app.tables["Recently Closed Tabs List"])
+    func testRecentlyClosedWebsiteOpen() {
+        openBookOfMozilla()
 
-        // Go to the default web site  and check whether the option is enabled
-        navigator.nowAt(LibraryPanel_History)
-        navigator.goto(HomePanelsScreen)
-        navigator.openURL(path(forTestPage: "test-mozilla-book.html"))
-        navigator.nowAt(BrowserTab)
-        navigator.goto(BrowserTabMenu)
-        // Workaround to bug 1508368
-        navigator.goto(LibraryPanel_Bookmarks)
-        navigator.goto(HomePanelsScreen)
-        navigator.performAction(Action.OpenNewTabFromTabTray)
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
+        // The tab, which is still opened, is not included in the "Recently Closed Tabs List"
+        navigator.nowAt(HomePanelsScreen)
         navigator.goto(LibraryPanel_History)
-        navigator.goto(HistoryRecentlyClosed)
-        waitForNoExistence(app.tables["Recently Closed Tabs List"])
-        navigator.nowAt(LibraryPanel_History)
-        navigator.goto(HomePanelsScreen)
+        waitForExistence(app.tables[HistoryPanelA11y.tableView])
+        XCTAssertFalse(app.tables.cells.staticTexts[bookOfMozilla["label"]!].exists)
+        XCTAssertTrue(app.tables[HistoryPanelA11y.tableView].staticTexts[emptyRecentlyClosedMesg].exists)
+    }
 
-        // Now go back to default website close it and check whether the option is enabled
-        navigator.openURL(path(forTestPage: "test-mozilla-book.html"))
-        waitUntilPageLoad()
-        waitForTabsButton()
-        navigator.goto(TabTray)
-        navigator.performAction(Action.AcceptRemovingAllTabs)
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
-        navigator.goto(BrowserTabMenu)
+    func testRecentlyClosedWebsiteClosed() {
+        openBookOfMozilla()
+
+        closeFirstTabByX()
+
+        // On regular mode, the closed tab is listed in "Recently Closed Tabs List"
+        navigator.nowAt(HomePanelsScreen)
```

Comment:
You get here but with the keyboard shown, you need these two actions so that the test passes:
```
navigator.performAction(Action.CloseURLBarOpen)
navigator.nowAt(NewTabScreen)
```